### PR TITLE
Better titles for articles

### PR DESCRIPTION
**Pull request by Rafase282**
Removes `-` and `:` from titles and also makes sure titles are more self explanatory.

### CODE REVIEW

File: CSS-Framework-Bootstrap.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Bootstrap
+# CSS Franework Bootstrap
```
Comment (abhisekp) [inline CSS-Framework-Bootstrap.md]: I :heart: "Franework" :wink:

Comment (shaginthecloud) [inline CSS-Framework-Bootstrap.md]: I could have done better

**Comment by shaginthecloud** [comment]
I wish I would have prepared my self better world friend

**Comment by Rafase282** [comment]
What do you mean @shaginthecloud ?

**Comment by shaginthecloud** [comment]
Spelled wrong

**Comment by shaginthecloud** [comment]
Sorry I couldn't have done better

**Comment by abhisekp** [comment]
@shaginthecloud you may hover over the line numbers and a [**`+`**]() symbol appears which you can click to add a note/comment in the corresponding line. :smiley:

File: Challenge-Get-Geo-Location-Data.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Challenge Get Geo-location Data
+# Challenge Get Geo location Data
```
Comment (theflametrooper) [inline Challenge-Get-Geo-Location-Data.md]: ~~Geo location~~ Geo Location

**Comment by theflametrooper** [comment]
### **LGTM :+1:**

Just make that quick change @Rafase282

**Comment by Rafase282** [comment]
All done @theflametrooper

**Pull request merged**